v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/BigQueryConverters.java

```diff
@@ -88,6 +93,49 @@
 
   private static final JsonFactory JSON_FACTORY = Transport.getJsonFactory();
 
+
+  /** Converts from the BigQuery Avro format into Bigtable mutation. */
+  @AutoValue
+  public abstract static class AvroToMutation implements SerializableFunction<SchemaAndRecord, Mutation> {
+
+    public abstract String columnFamily();
+
+    public abstract String rowkey();
+
+    /** Builder for AvroToEntity. */
+    @AutoValue.Builder
+    public abstract static class Builder {
+
+      public abstract Builder setColumnFamily(String value);
+
+      public abstract Builder setRowkey(String rowkey);
+
+      public abstract AvroToMutation build();
+    }
+
+    public static Builder newBuilder() {
+      return new AutoValue_BigQueryConverters_AvroToMutation.Builder();
+    }
+
+    public Mutation apply(SchemaAndRecord record) {
+      GenericRecord row = record.getRecord();
+      String rowkey = row.get(rowkey()).toString();
+      Put put = new Put(Bytes.toBytes(rowkey));
+
+      List<TableFieldSchema> columns = record.getTableSchema().getFields();
+      for (TableFieldSchema column : columns) {
+        String columnName = column.getName();
+        String columnValue = row.get(columnName).toString();
+        // TODO: handle other types and column families
```

Comment:
Could you please add your GitHub username here? It'll make it easier to track and get context.